app/js/models/todo.js

```diff
@@ -0,0 +1,12 @@
+define([
+  'backbone'
+], function (Backbone) {
+  var ToDo = Backbone.Model.extend({
+    defaults: {
+      title: '',
+      completed: false
```

Comment:
For the current task the `completed` attribute was not required, same for `title`